src/frontend/utils/test/validators.test.ts

```diff
@@ -44,6 +44,7 @@ describe('utils/validators', () => {
     const mockBarn = {
         personIdent: '12345678930',
         fødselsdato: '2000-05-17',
+        dødsfallDato: '2020-12-12',
```

Comment:
Synes det rart er med et felles mock objekt der barnet er dødt 🤔 
Kunne dette bli gjort mer eksplisitt med en fixture?
Typ: https://michalzalecki.com/fixtures-the-way-to-manage-sample-and-test-data/

```
const grunnlagPersonFixture = (overstyrendeProps: Partial<IGrunnlagPerson> = {}) => {
        const defaults: IGrunnlagPerson = {
            personIdent: '12345678930',
            fødselsdato: '2000-05-17',
            type: PersonType.BARN,
            kjønn: kjønnType.KVINNE,
            navn: 'Mock Barn',
            målform: Målform.NB,
        };

        return { ...defaults, ...overstyrendeProps };
    };
``` 

Så kunne man brukt `grunnlagPersonFixture({dødsfallDato: '2020-12-12'})` i testen. Da hadde det blitt litt mer tydelig at det var en vanlig person, men som var dødt

Reply:
Enig!

Jeg endrer på det nå 😊